priam/src/main/java/com/netflix/priam/tuner/GCTuner.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2016 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.netflix.priam.tuner;
+
+import javax.inject.Singleton;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * List of Garbage collection parameters for CMS/G1GC. This list is used to automatically enable/disable configurations, if found in jvm.options.
+ * Created by aagrawal on 8/23/17.
+ */
+@Singleton
+public class GCTuner {
+//    private Set<String> commonOptions = new HashSet<>(Arrays.asList(
+//            "-XX:ParallelGCThreads",
+//            "-XX:ConcGCThreads",
+//            "-XX:+ParallelRefProcEnabled",
+//            "-XX:+AlwaysPreTouch",
+//            "-XX:+UseTLAB",
+//            "-XX:+ResizeTLAB",
+//            "-XX:-UseBiasedLocking"
+//    ));
+
+    private static final Set<String> cmsOptions = new HashSet<>(Arrays.asList(
```

Comment:
Won't we have any thing in common?

Reply:
We would, but the idea is not to touch anything in common. Configuring GC will only enable/disable the parameters which are specific to GC.